### DISCUSSION TITLE
New native app suggestion.

### DISCUSSION THREAD

**Discussion by manu20247** [elementary/os#705] [Ideas]
Hello, first of all I want to thank you for the excellent work you do.  Well, I really miss having a native word processor on elementary OS, I really miss it, and I don't like using libreoffice, the interface is ugly, visually polluted and cumbersome to run.  There is an application that was created to process texts made especially for elementary students, you can fork the code.  I will send the access link.  Please fork Gnumeric too, we really need a spreadsheet and text processor.  but that matches the interface, bringing integration and elegance.  We lack these applications and we are orphans because we cannot find them natively.  Thank you very much, I would like to see this app linked below still in version 8 of Elementary OS if possible.

Link of the App = https://github.com/ryonakano/writer

![d5t9wcs-9ff60bd1-1784-48d7-82c5-b2899c3ab04e](https://github.com/elementary/os/assets/163211417/886436b8-5379-47de-8d27-4280ea8b1435)

**Comment by teamcons** [comment]
The app you linked is from one of elementary devs
The team does not have the resources for this

**Reply by MediaElf** [reply]
Hi, I am interested in @manu20247's suggestion and I fully support it, however I did not understand your response

**Reply by teamcons** [reply]
> Hi, I am interested in @manu20247's suggestion and I fully support it, however I did not understand your response

The app they speak of, in the screenshot, was made by Ryo, an elementary OS developer.

Right now the development team is too small to consider things like that. We need more people to make the ideas into something.

So it stays as a third party app

**Reply by MediaElf** [reply]
Well, how can you add it to the AppStore so that every dummies can install it?

**Reply by stsdc** [reply]
You need to ask @ryonakano to publish it.

**Reply by teamcons** [reply]
Pretty much. But considering, a small change in december aside, that the app isnt actively worked on since summer 2023, tells me that either Ryo is busy with something else, or lost interest

I dont have time to maintain it, theres already a lot to do with my own apps, elementary OS, and the rest of my life. And i suspect it is the same for Ryo or the others

**Reply by ryonakano** [reply]
Yes, what @teamcons wrote above is correct. I forked Writer from Launchpad for a while ago and IIRC it was my first attempt to develop my own app but I'm almost no longer interested in developing it. Unfortunately developing a word processor seems to be overwhelming for me due to lack of my knowledge.

Also, the above image @manu20247 is a mockup of Writer maybe originally from [spiceofdesign on DeviantArt](https://www.deviantart.com/spiceofdesign/art/Writer-Concept-351501580). FYI I attach a screenshot of Writer built in [the latest main branch](https://github.com/ryonakano/writer/commit/f0c6648e38b78f36becf766eff1231fa909cde7a), which is quite inferior to the mockup.

![image](https://github.com/user-attachments/assets/1d98520f-7876-4888-bc99-396e16f3140f)

**Reply by teamcons** [reply]
maybe we could submit it to appcenter, and let it stay there ? Maybe that would add visibility and later on interest from others to maintain it ?

**Reply by ryonakano** [reply]
Hmm I'm pretty not eager to publish apps that doesn't do almost things that users would expects……

**Reply by teamcons** [reply]
Okay fair

**Comment by ryonakano** [comment]
Closing as this is a community development app and we have no plan to ship it as a default application on Pantheon at the moment.
